src/main/java/fr/insee/bpm/metadata/reader/lunatic/LunaticReader.java

```diff
@@ -149,7 +138,13 @@ public static MetadataModel getMetadataFromLunatic(InputStream lunaticFile) {
 			rootNode = JsonReader.read(lunaticFile);
 			List<String> variables = new ArrayList<>();
 			JsonNode variablesNode = rootNode.get(VARIABLES);
-			variablesNode.forEach(newVar -> variables.add(newVar.get("name").asText()));
+			variablesNode.forEach(newVar -> {
+				String varName = newVar.get("name").asText();
```

Comment:
use LunaticUtils.getVariableName

Reply:
It won't work here, the node of a variable does not have a RESPONSE attribute.